concepticondata/conceptlists/Tolmie-1884-211.tsv

```diff
@@ -0,0 +1,212 @@
+Tolmie-1884-211-60	60	Canoe	1970	CANOE
+Tolmie-1884-211-61	61	Moccasins		
+Tolmie-1884-211-62	62	Pipe	1124	PIPE
+Tolmie-1884-211-63	63	Tobacco	974	TOBACCO
+Tolmie-1884-211-64	64	Sky	1732	SKY
+Tolmie-1884-211-65	65	Sun	1343	SUN
+Tolmie-1884-211-66	66	Moon	1313	MOON
+Tolmie-1884-211-67	67	Star	1430	STAR
+Tolmie-1884-211-68	68	Day	1225	DAY (NOT NIGHT)
+Tolmie-1884-211-69	69	Night	1233	NIGHT
+Tolmie-1884-211-70	70	Morning	1339	MORNING
+Tolmie-1884-211-71	71	Evening	1629	EVENING
+Tolmie-1884-211-72	72	Spring	1537	SPRINGTIME
+Tolmie-1884-211-73	73	Summer	1342	SUMMER
+Tolmie-1884-211-74	74	Autumn	1276	AUTUMN
+Tolmie-1884-211-75	75	Winter	1471	WINTER
+Tolmie-1884-211-76	76	Wind	960	WIND
+Tolmie-1884-211-77	77	Thunder	1150	THUNDER
+Tolmie-1884-211-78	78	Lightning	1654	LIGHTNING
+Tolmie-1884-211-79	79	Rain	658	RAIN (PRECIPITATION)
+Tolmie-1884-211-80	80	Snow	784	SNOW
+Tolmie-1884-211-81	81	Fire	221	FIRE
+Tolmie-1884-211-82	82	Water	948	WATER
+Tolmie-1884-211-83	83	Ice	617	ICE
+Tolmie-1884-211-84	84	Earth, land		
+Tolmie-1884-211-85	85	Sea	1474	SEA
+Tolmie-1884-211-86	86	River	666	RIVER
+Tolmie-1884-211-87	87	Lake	624	LAKE
+Tolmie-1884-211-88	88	Valley	921	VALLEY
+Tolmie-1884-211-89	89	Prairie		
+Tolmie-1884-211-90	90	Hill, mountain	2118	MOUNTAIN OR HILL
+Tolmie-1884-211-91	91	Island	622	ISLAND
+Tolmie-1884-211-92	92	Stone, rock	2125	STONE OR ROCK
+Tolmie-1884-211-93	93	Salt	1274	SALT
+Tolmie-1884-211-94	94	Iron	621	IRON
+Tolmie-1884-211-95	95	Forest	420	FOREST
+Tolmie-1884-211-96	96	Tree	906	TREE
+Tolmie-1884-211-97	97	Wood	1803	WOOD
+Tolmie-1884-211-98	98	Leaf	628	LEAF
+Tolmie-1884-211-99	99	Bark	1204	BARK
+Tolmie-1884-211-100	100	Grass	606	GRASS
+Tolmie-1884-211-101	101	Pine	1740	PINE
+Tolmie-1884-211-102	102	Maize	506	MAIZE
+Tolmie-1884-211-103	103	Squash	864	PUMPKIN
+Tolmie-1884-211-104	104	Flesh, meat	2615	FLESH OR MEAT
+Tolmie-1884-211-105	105	Dog	2009	DOG
+Tolmie-1884-211-106	106	Buffalo	934	BUFFALO
+Tolmie-1884-211-107	107	Bear (black)		
+Tolmie-1884-211-108	108	Wolf	522	WOLF
+Tolmie-1884-211-109	109	Fox	1312	FOX
+Tolmie-1884-211-110	110	Deer	1936	DEER
+Tolmie-1884-211-111	111	Elk	1772	ELK
+Tolmie-1884-211-112	112	Beaver	1194	BEAVER
+Tolmie-1884-211-113	113	Rabbit, hare	2345	LEPORID (RABBIT OR HARE)
+Tolmie-1884-211-114	114	Tortoise	895	TORTOISE
+Tolmie-1884-211-115	115	Horse	615	HORSE
+Tolmie-1884-211-116	116	Fly	1504	FLY (INSECT)
+Tolmie-1884-211-117	117	Mosquito	1509	MOSQUITO
+Tolmie-1884-211-118	118	Snake	730	SNAKE
+Tolmie-1884-211-119	119	Rattlesnake	871	RATTLESNAKE
+Tolmie-1884-211-120	120	Bird	937	BIRD
+Tolmie-1884-211-121	121	Egg	744	EGG
+Tolmie-1884-211-122	122	Feathers	1201	FEATHER
+Tolmie-1884-211-123	123	Wings	1257	WING
+Tolmie-1884-211-124	124	Goose	1187	GOOSE
+Tolmie-1884-211-125	125	Duck (mallard)	1360	DUCK
+Tolmie-1884-211-126	126	Turkey	2386	TURKEY
+Tolmie-1884-211-127	127	Pigeon	1853	DOVE
+Tolmie-1884-211-128	128	Fish	227	FISH
+Tolmie-1884-211-129	129	Salmon	944	SALMON
+Tolmie-1884-211-130	130	Sturgeon		
```

Comment:
This will require some digging, but I'd be inclined to add the Sturgeon, as it is an example of flora and fauna. It should be the typical representative in this area, of course, as there are too many species of the Stör.